test/text/test_cell.py

```diff
@@ -316,7 +316,7 @@ def test_cell_deprecated_txt_arg():
         DeprecationWarning, match='The parameter "txt" has been renamed to "text"'
     ):
         # pylint: disable=unexpected-keyword-arg
-        pdf.cell(txt="Lorem ipsum Ut nostrud irure")
+        pdf.cell(text="Lorem ipsum Ut nostrud irure")
```

Comment:
It looks like I have misled you a bit by recommending to just search for instances of "txt=".

You also need to look at what the purpose of the test actually is. In this case (and the next three), the test explicitly checks if fpdf2 correctly produces a warning for the deprecated parameter.
For that to work, the "txt" needs to stay there. You can also see that the "incorrect" call is protected by a pylint control comment (that currently makes the test suite fail, because it has no purpose anymore).

So you'll have to revert the changes to those tests. The two fixes to the tutorial code are fine.

Reply:
> It looks like I have misled you a bit by recommending to just search for instances of "txt=".
> 
> You also need to look at what the purpose of the test actually is. In this case (and the next three), the test explicitly checks if fpdf2 correctly produces a warning for the deprecated parameter. For that to work, the "txt" needs to stay there. You can also see that the "incorrect" call is protected by a pylint control comment (that currently makes the test suite fail, because it has no purpose anymore).
> 
> So you'll have to revert the changes to those tests. The two fixes to the tutorial code are fine.

I have made changes now . Could you please take a look at the changes and let me know if you have any feedback?